### ISSUE TITLE
❓ Loading remote content if no tracker was detected 

### ISSUE THREAD

**Issue by Erik262** [apparition47/MailTrackerBlocker#109]
Hi folks,

Is there any option to load content automatically if no tracker was detected in the mail?

**Comment by apparition47** [comment]
No such option right now. Plugin assumes "Load Remote Content" is always enabled.

Would this be a useful feature?

**Comment by toobuntu** [comment]
> No such option right now. Plugin assumes "Load Remote Content" is always enabled.
> 
> Would this be a useful feature?

Yes. It would also be useful if a tracker was found but was "defeated" or sanitized by the plugin. 

To clarify, will this plugin still block trackers if "Load Remote Content" is disabled by default and only selectively enabled on a per-message basis?

**Comment by apparition47** [comment]
>To clarify, will this plugin still block trackers if "Load Remote Content" is disabled by default and only selectively enabled on a per-message basis?

Yes
